File: LeetCode/Easy/0389-find-the-difference/0389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int counter[26]{0};
        char ret = 'a';
        for (int i = 0; i < s.size(); ++i){
            ++counter[s[i] - 'a'];
        }
        for (int i = 0; i < t.size(); ++i){
            counter[t[i]-'a']--;
        }
        for(int i = 0; i < 26; ++i){
            if (counter[i] != 0) {
                ret = i + 'a';
                break;
            }
        }
        return ret;
    }
};

File: LeetCode/Easy/0389-find-the-difference/0389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int[] counter = new int[26];
        char ret = 'a';
        for (char v : s.toCharArray()){
            ++counter[v - 'a'];
        }
        for (char v : t.toCharArray()){
            counter[v-'a']--;
        }
        for(int i = 0; i < counter.length; ++i){
            if (counter[i] != 0) {
                ret = (char) (i + 'a');
            }
        }
        return ret;
    }
}